src/components/data/data-utils.js

```diff
@@ -115,8 +118,7 @@ export const renderDataCell = (attributeValue, workspace) => {
 
   const renderCell = datum => {
     const stringDatum = Utils.convertValue('string', datum)
-
-    return isUri(datum) ? h(UriViewerLink, { uri: datum, workspace }) : stringDatum
+    return isUri(datum, workspace.googleProject) ? h(UriViewerLink, { uri: datum, workspace }) : stringDatum
```

Comment:
The preferred way to check if the workspace is a Google workspace is to use `isGoogleWorkspace` from `workspace-utils` (and similarly `isAzureWorkspace`).

Reply:
Updated this PR to use those functions!